modules/standard/BigInteger.chpl

```diff
@@ -2349,21 +2349,27 @@ module BigInteger {
   // divexact
 
 /*
-Computes ``n/d`` and stores the result in ``bigint`` instance.
+Computes ``numer/denom`` and stores the result in ``bigint`` instance.
 
-``divexact`` is optimized to handle cases where ``n/d`` results in an integer.
-When ``n/d`` does not produce an integer, this method may produce incorrect results.
+``divexact`` is optimized to handle cases where ``numer/denom`` results in an integer.
+When ``numer/denom`` does not produce an integer, this method may produce incorrect results.
 
-:arg n: numerator
+:arg numer: numerator
 
-:type n: bigint
+:type numer: bigint
 
-:arg d: denominator
+:arg denom: denominator
 
-:type d: bigint
+:type denom: bigint
```

Comment:
Nice job updating the documentation comment!  Unfortunately, adding the warning below in a separate comment will cause this comment to not be generated in the documentation (chpldoc only uses the closest comment to a symbol when determining documentation) but I think we want it to be attached to the new versions anyways

Reply:
Oh sorry, I was not aware of this, Can you please guide me on how I can proceed with this?

Reply:
Sorry, I should have been more clear - I think the best solution is to just move the main comment to the new version of `divexact`.

Reply:
Sounds good, I have moved the comment now.